tests/test_unit/test_cli/test_build_deploy_snapshots/complete_org_alpha_flags.yaml

```diff
@@ -140,7 +140,6 @@ InFieldCDMLocationConfig:
     threeD: true
     trends: true
     workorders: true
-  instanceType: node
```

Comment:
These are not necessary to include when dumping Infield configs. When we do a request against CDF, they are made into an InstanceRequest which dumps the instanceType.